daemon/kill.go

```diff
@@ -105,6 +115,9 @@ func (daemon *Daemon) Kill(container *container.Container) error {
 		return errNotRunning{container.ID}
 	}
 
+	// FIXME(vdemeester) should we disable restartpolicy for SIGKILL or not ?
```

Comment:
So, this is only in the case of a _shutdown_, correct? I think a shutdown should not influence the restart policy, independent of the signal that was used (i.e. if the container was running, and it's killed as part of the shutdown, the _intent_ was to have it running, so `--restart=unless-stopped` still applies).